app/templates/components/article-wrapper.hbs

```diff
@@ -13,10 +13,16 @@
 {{/if}}
 
 {{#if hasFeaturedVideo}}
-	{{article-featured-video
-			model=model.featuredVideo
-			toggleSiteHeadShadow=(action 'toggleSiteHeadShadow')
-	}}
+    {{#if isJWPlayer}}
+        {{article-featured-video-jwplayer
+            model=model.featuredVideo
+        }}
+    {{else}}
+        {{article-featured-video-ooyala
+            model=model.featuredVideo
+            toggleSiteHeadShadow=(action 'toggleSiteHeadShadow')
```

Comment:
What is it? Don't we need it in JW? :D

Reply:
it's related to onscroll, we don't have onscroll on mobile for now